crates/goose-cli/src/prompt/renderer.rs

```diff
@@ -171,12 +171,24 @@ pub fn default_response_renderer(tool_response: &ToolResponse, theme: &str) {
 
 pub fn default_print_request_header(call: &ToolCall) {
     // Print the tool name with an emoji
-    let parts: Vec<_> = call.name.split("__").collect();
+
+    // use rsplit to handle any prefixed tools with more underscores
+    // unicode gets converted to underscores during sanitization
+    let parts: Vec<_> = call.name.rsplit("__").collect();
```

Comment:
could be worth making this a helper function `prefixed_name -> (system_name, tool_name)`

Reply:
from within the renderer it doesn't have the same capability to determine the `system_name`, as we do within the capabilities change (i.e. the clients map) - this is done best effort since `__` is allowed basically in both sides of the naming